src/main/java/seedu/address/logic/parser/AddModuleCommandParser.java

```diff
@@ -24,7 +24,12 @@
      */
     public AddModuleCommand parse(String args) throws ParseException {
         String[] tokenizedArgs = ParserUtil.tokenize(args);
-        argsWithBounds(tokenizedArgs, 4, 5);
+        String inputSuggestion = "\nInput order: c_add MODULE_CODE YEAR SEMESTER CREDIT GRADE"
+                + "\nExample 1: c_add CS2103 2 1 A+"
+                + "\nIf you haven't taken the module before, you can omit the grade:"
+                + " Example 2: c_add CS2103 2 1 ";
+
+        argsWithBounds(tokenizedArgs, 4, 5, inputSuggestion);
```

Comment:
I think you can use `AddModuleCommand.MESSAGE_USAGE` instead of creating a new `inputSuggestion`. If the MESSAGE_USAGE is not detailed enough then we can make `MESSAGE_USAGE` to be more detailed.
```suggestion
        argsWithBounds(tokenizedArgs, 4, 5, AddModuleCommand.MESSAGE_USAGE);
```